src/panels/CreateClusterPanel.ts

```diff
@@ -203,25 +227,52 @@ async function createCluster(
     }
 }
 
-function getManagedClusterSpec(location: string, name: string): ManagedCluster {
-    return {
-        addonProfiles: {},
-        location: location,
-        identity: {
-            type: "SystemAssigned",
-        },
-        agentPoolProfiles: [
-            {
-                name: "nodepool1",
-                type: "VirtualMachineScaleSets",
-                count: 3,
-                enableNodePublicIP: true,
-                mode: "System",
-                osSKU: "AzureLinux",
-                osType: "Linux",
-                vmSize: "Standard_DS2_v2",
-            },
-        ],
-        dnsPrefix: `${name}-dns`,
+function getManagedClusterSpec(clusterSpec: ClusterSpec, preset: string): Deployment {
```

Comment:
💡☕️ Thoughts 💭 if this sounds super cool, shall we heavily document this function and  add some clarity as to even this says the aspects enterprise and various other versions it is not the full blown “prod”, “enterprise” et al aka portal obviously has dense options to make it more u/x driven.

This is super cool and what personally I think this does is to provide that seem less experience which leads to the IDE to stay as it is and not scattered which leads to the context changes which I think leads to break for the focused workflow. ❤️🙏

Reply:
I agree, right now we have only one preset dev/test, I think we can document heavily once we introduce the other presets in the future releases.

Reply:
@Tatsinnit  agree ?

Reply:
👍 absolutely with clarity that this feature is quick create cluster feature from users favourite editor aka vscode and simplifies the experience. (Which means portal will have full blown functionality available if anyone keen)